sources/models/src/modeled_types/shared.rs

```diff
@@ -558,3 +558,44 @@ impl TryFrom<&str> for Lockdown {
 }
 
 string_impls_for!(Lockdown, "Lockdown");
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct ValidContainerMode {
```

Comment:
```suggestion
pub struct ContainerMode {
```
I regret the `Valid` prefix on `Base64`... I was trying to disambiguate from the library name it's wrapping, but any type we create should be "valid."  Inventions of our own, in particular, don't have any other state.  A container mode can be "off," "once," or "always," because that's how it's defined, and anything else isn't a container mode at all - no need to say valid.

Reply:
Done